docs/data/base/components/button/button.md

```diff
@@ -1,17 +1,49 @@
 ---
 product: base
-title: React button unstyled
+title: Unstyled React button
+components: ButtonUnstyled
 githubLabel: 'component: button'
+waiAria: https://www.w3.org/TR/wai-aria-practices/#button
 ---
 
-# Button unstyled
+# Unstyled button
 
 <p class="description">Buttons allow users to take actions, and make choices, with a single tap.</p>
 
 ## Basic usage
 
+```js
+import ButtonUnstyled from '@mui/base/ButtonUnstyled';
+```
+
+{{"demo": "UnstyledButtonsSimple.js"}}
+
 ## Customizing the root element
 
+By default, the `ButtonUnstyled` component renders a native `button` HTML element.
+You can override this by setting the `component` or `components.Root` prop.
+If you provide a non-interactive element, such as a `span`, the `ButtonUnstyled` component will automatically add the necessary accessibility attributes.
+
+{{"demo": "UnstyledButtonsSpan.js"}}
+
+Compare the attributes on the `span` with the `button` from the previous demo.
+
 ## Complex customization
 
-## `useButton` hook
+In addition to HTML elements, you can also use SVGs with the 'ButtonUnstyled` component.
```

Comment:
```suggestion
In addition to HTML elements, you can also use SVGs with the `ButtonUnstyled` component.
```

...or remove the backticks. We should decide and be consistent about surrounding the component name with backticks. I'd say it's not necessary to show component names as code. I would use them only for prop and parameter names and values.
cc @samuelsycamore